### PR TITLE
Update Zenhub api endpoint

### LINKED ISSUE #10611: [BUG] The Github > Socrata ETL is failing with a Zenhub SSL error
[This DAG](https://github.com/cityofaustin/atd-airflow/blob/master/dags/atd_service_bot_issues_to_socrata.py) is [failing](https://airflow.austinmobility.io/log?task_id=dts_github_to_socrata&dag_id=atd_service_bot_github_to_socrata_production&execution_date=2022-10-26T18%3A21%3A34.000419%2B00%3A00).

```
[2022-10-26 18:24:20,161] {{docker_operator.py:242}} INFO - INFO:root:Fetching zenhub data...
....
ssl.SSLCertVerificationError: ("hostname 'api.zenhub.io' doesn't match 'raptor-admin.zenhub.com'",)
```

According to Zenhub's [legacy REST API docs](https://github.com/ZenHubIO/API#root-endpoint), we may not be using the right endpoint.

### PR DESCRIPTION
**Pull request by johnclary**
Fixes https://github.com/cityofaustin/atd-data-tech/issues/10611

Steps to test:
The [Socrata dataset](https://data.austintexas.gov/City-Government/ATD-Data-Tech-Services-Issues/rzwg-fyv8) has been updated from my local running of this script. You can verify by sorting on "Created" column and observing issues created today.

To test locally:
1. grab the service bot env file from our credential store
2. checkout this branch
3. run
```
docker run -it --rm --env-file env_file -v your/path/to/this/repo/atd-service-bot:/app atddocker/atd-service-bot:production python issues_to_socrata.py
```

### CODE REVIEW

File: from_csv.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 
 from github import Github
 
+
```
Comment (johnclary) [inline from_csv.py:33]: linting

File: issues_to_socrata.py [hunk 2]
```diff
@@ -73,7 +73,7 @@ def convert_timestamps(issues):
 
 
 def get_zenhub_metadata(workspace_id, token, repo_id, timeout=60):
-    url = f"https://api.zenhub.io/p2/workspaces/{workspace_id}/repositories/{repo_id}/board"
+    url = f"https://api.zenhub.com/p2/workspaces/{workspace_id}/repositories/{repo_id}/board"
```
Comment (mddilley) [inline issues_to_socrata.py:76]: 🙌  I definitely found a couple Gists with the .io endpoint yesterday so it had to have been documented somewhere before.

**Review by mddilley — APPROVED** [review]
I see current issues in the Socrata dataset and also tested locally. Everything works great! 🎉

File: issues_to_socrata.py (hunk 2 above)
Comment (johnclary) [reply issues_to_socrata.py:76]: would have been nice to get a heads up! https://github.com/ZenHubIO/API/commit/3a5405bc058e64e1c3b79f79e85e121db54012e9

**Comment by johnclary** [comment]
thanks @mddilley!

**Comment by Charlie-Henry** [comment]
If you got this in docker already I can Prefect-ify this pretty easily if you like 😉

**Comment by johnclary** [comment]
@Charlie-Henry if you're offering it would be lovely of you to migrate this. there are actually three DAGs that use this image (all prefixed with `atd_serivce_bot`. i'll make an issue 😁

**Pull request merged**